.travis.yml

```diff
@@ -25,7 +25,7 @@ install:
   - eval "$(curl -sL https://swiftenv.fuller.li/install.sh)"
   
   # clone and build libmongoc
-  - git clone -b r1.10 https://github.com/mongodb/mongo-c-driver /tmp/libmongoc
+  - git clone -b r1.12 https://github.com/mongodb/mongo-c-driver /tmp/libmongoc
```

Comment:
Should this be in its own commit (related to bumping the dependency elsewhere)? It doesn't look like something that should end up in a single squashed commit from merging this PR.

Reply:
Sure, I will bump it in a separate commit to master.

Reply:
[done](https://github.com/mongodb/mongo-swift-driver/commit/57a799eef066a37371eba47874d14de6547cde69)